rental.rb

```diff
@@ -0,0 +1,16 @@
+require './person'
+require './book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.visits << self
+
+    @book = book
+    book.visits << self
```

Comment:
Can we make this to be `book.rentals << self` since we working with rentals. The rental is adding itself to the @book. Nice one though.